tests/RobotFramework/tests/cumulocity/configuration/legacy_config_operation/tedge-configuration-plugin.toml

```
@@ -0,0 +1,5 @@
+files = [
+    { path = '/etc/tedge/tedge.toml', user = 'tedge', group = 'tedge', mode = 0o444 },
+    { path = '/etc/tedge/system.toml', type = 'system.toml', user = 'tedge', group = 'tedge', mode = 0o444 },
+    { path = '/etc/legacy.txt', type = 'legacy', user = 'tedge', group = 'tedge', mode = 0o644 },
```

The term *legacy* sounds a bit weird when out of its context, i.e. the context of Cumulocity.

```suggestion
    { path = '/etc/legacy.txt', type = 'default', user = 'tedge', group = 'tedge', mode = 0o644 },
```

---

resolved by 2b1fd5e54014d134befacf2ca7aef7626f7995d0